.changeset/breezy-cats-grab.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': patch
+'@astrojs/markdoc': patch
+'@astrojs/mdx': patch
```

Comment:
Since `@astrojs/mdx` is updated to use the new APIs from next Astro's patch (so previous version won't work), should we mark this a minor to make a breaking release?

Reply:
I'm a little confused why `@astrojs/mdx` is even included in this changeset that is all about `@astrojs/markdoc`. 

If you could create multiple changesets that describe what actually changed in each package that would be ideal.

Reply:
Discussed on discord! Sounds like adding a legacy check to make this non-breaking would be the best path 👍